create_pr/create_pr.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+#  Copyright (c) 2022 Ecco Sneaks & Data
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -Eeuo pipefail
+
+git add .
+git commit -m "Updating $PROJECT_NAME to $PROJECT_VERSION"
+git push --set-upstream origin "update-$PROJECT_NAME-$PROJECT_VERSION"
+
+if [[ "$DEPLOY_ENVIRONMENT" != "test" ]]; then
```

Comment:
i think in this action we should just have a flag that allows pr merge. And the `if` logic should be inside the workflow